src/components/generic/themed-calendar.tsx

```diff
@@ -1,17 +1,92 @@
 import React from "react";
 import { Calendar } from "react-native-calendars";
+import { MarkingProps } from "react-native-calendars/src/calendar/day/marking";
+import { DateData } from "react-native-calendars/src/types";
+import { ExposureByDate } from "../../types/types";
+import StatisticsUtils from "../../utils/statistics-utils";
+
+
+/**
+ * Component props
+ */
+interface Props {
+  selectedDate: Date;
+  monthExposures: ExposureByDate[]
+  onDatePress: (selectedData: DateData) => void;
+  onMonthChange?: ((date: DateData) => void);
```

Comment:
```suggestion
  onMonthChange?: (date: DateData) => void;
```
SHould it be like that?